frame/contracts/src/wasm/mod.rs

```diff
@@ -406,6 +409,10 @@ mod tests {
 			}
 			Ok(result)
 		}
+		fn set_code_hash(&mut self, hash: CodeHash<Self::T>) -> Result<(), DispatchError> {
+			self.storage.insert(StorageKey::from(hash), vec![1, 2, 3]);
+			Ok(())
+		}
```

Comment:
This feels hacky. We should add a new field to the mock ext which records all set code hashes (just like we do for other stuff).